Src/Support/Google.Apis.Auth.Tests/OAuth2/ServiceAccountCredentialTests.cs

```diff
@@ -686,17 +686,29 @@ public void WithUseJwtAccessWithScopes()
             Assert.True(credentialWithJwtFlag.UseJwtAccessWithScopes);
         }
 
-        [Fact]
-        public async Task FetchesOidcToken()
+        public static TheoryData<string, string, string, string> OidcData => new TheoryData<string, string, string, string>
+        {
+            // universe domain, token URL, SA id, expected OIDC URL
+            { null, "http://will.be.ignored", "MyId", "http://will.be.ignored/" },
```

Comment:
"will.be.ignored" sounds wrong if we're later asserting that we've actually made a call to it. (I'd expected tests which demonstrated the more radically different paths taken here, but maybe I just don't understand.)